docs/DeveloperGuide.md

```diff
@@ -135,6 +135,10 @@ The Sequence Diagram below illustrates the interactions within the `Logic` compo
 <div markdown="span" class="alert alert-info">:information_source: **Note:** The lifeline for `DeleteCommandParser` and `DeleteCommand` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
 </div>
 
+The Activity Diagram below accompanies the `DeleteCommand`.
+
+![Activity Diagram of Delete Command](images/DeleteActivityDiagram.png)
```

Comment:
text on the left side crosses multiple lines, can be quite confusing which it belongs to